experiments/configs/hugging-face/eval_configs/eval_pipeline_config.yaml

```diff
@@ -2,6 +2,8 @@ model: hf-causal
 model_args: "pretrained=/fsx/proj-chemnlp/experiments/checkpoints/finetuned/full_1b/checkpoint-final"
 # model_args: "pretrained=EleutherAI/pythia-1b"
 tasks: "hendrycksTest-college_biology,hendrycksTest-college_chemistry,hendrycksTest-college_mathematics,hendrycksTest-college_physics,hendrycksTest-high_school_mathematics,hendrycksTest-high_school_biology,hendrycksTest-high_school_chemistry,hendrycksTest-high_school_physics"
+batch_size: 8
```

Comment:
does > 8 lead to memory errors on the GPU?

Reply:
It was 8 because I copied it from the EleutherAI args examples but I just checked and 12 runs fine and is faster so I increased it